src/pathways/pathways.module.ts

```
@@ -14,6 +14,7 @@ import { InterestsModule } from './interests/interests.module';
 import { LmsClientService } from './common/services/lms-client.service';
 import { StorageModule } from '../storage/storage.module';
 import { CacheModule } from '../cache/cache.module';
+import { NotificationRequest } from '../common/utils/notification.axios';
```

ESLint

[error] Replace `'../common/utils/notification.axios'` with `"../common/utils/notification.axios"`

(prettier/prettier)